cli/cmd/profile.go

```diff
@@ -68,6 +69,7 @@ func buildConfig(namespace, service string) *templateConfig {
 		ControlPlaneNamespace: controlPlaneNamespace,
 		ServiceNamespace:      namespace,
 		ServiceName:           service,
+		ClusterZone:           "svc.cluster.local",
```

Comment:
I am not familiar with the background for this change but should we be cautious to not assume that the ClusterZone is `svc.cluster.local`? This is related to #1720 and I was thinking maybe this could be another command line flag for the profile command or maybe there is a better way to handle cases were the base cluster domain is not `cluster.local`.

Reply:
Let's keep that configuration out of scope for now. There are a number of places where this string is hardcoded.  Ultimately, I think this setting should be discoverable (rather than configurable).